Index: ManagementService/src/main/resources/ log4jdbc.log4j2.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>log4jdbc.spylogdelegator.name=net.sf.log4jdbc.log.slf4j.Slf4jSpyLogDelegator\r\nlog4jdbc.dump.sql.maxlinelength=0\r\n
===================================================================
diff --git a/ManagementService/src/main/resources/ log4jdbc.log4j2.properties b/ManagementService/src/main/resources/ log4jdbc.log4j2.properties
--- a/ManagementService/src/main/resources/ log4jdbc.log4j2.properties	
+++ b/ManagementService/src/main/resources/ log4jdbc.log4j2.properties	
@@ -1,2 +1,3 @@
 log4jdbc.spylogdelegator.name=net.sf.log4jdbc.log.slf4j.Slf4jSpyLogDelegator
 log4jdbc.dump.sql.maxlinelength=0
+log4jdbc.drivers=org.mariadb.jdbc.Driver
